code/datums/outfits/outfit_admin.dm

```diff
@@ -18,222 +18,436 @@
 		I.icon_state = special_icon
 
 /datum/outfit/admin
-/datum/outfit/admin/spacegear
-	name = "Standard Space Gear"
 
-	uniform = /obj/item/clothing/under/color/grey
-	suit = /obj/item/clothing/suit/space
-	back = /obj/item/weapon/tank/jetpack/oxygen
-	shoes = /obj/item/clothing/shoes/black
-	head = /obj/item/clothing/head/helmet/space
-	mask = /obj/item/clothing/mask/breath
-	l_ear = /obj/item/device/radio/headset
-	id = /obj/item/weapon/card/id
+/datum/outfit/admin/syndicate
+	name = "Syndicate Agent"
 
-/datum/outfit/admin/spacegear/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	uniform = /obj/item/clothing/under/syndicate
+	back = /obj/item/weapon/storage/backpack
+	belt = /obj/item/weapon/storage/belt/utility/full/multitool
+	gloves = /obj/item/clothing/gloves/combat
+	shoes = /obj/item/clothing/shoes/combat
+	l_ear = /obj/item/device/radio/headset/syndicate
+	id = /obj/item/weapon/card/id/syndicate
+	r_pocket = /obj/item/device/radio/uplink
+	backpack_contents = list(
+		/obj/item/weapon/storage/box/engineer = 1,
+		/obj/item/device/flashlight = 1,
+		/obj/item/weapon/card/emag = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/syndidonkpocket = 1
+	)
+
+	var/id_icon = "syndie"
+	var/id_access = "Syndicate Operative"
+	var/uplink_uses = 20
+
+/datum/outfit/admin/syndicate/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	. = ..()
 	if(visualsOnly)
 		return
 
-	if(istype(H.back, /obj/item/weapon/tank/jetpack))
-		var/obj/item/weapon/tank/jetpack/J = H.back
-		J.turn_on()
-		J.toggle_internals(H)
-
 	var/obj/item/weapon/card/id/I = H.wear_id
 	if(istype(I))
-		apply_to_card(I, H, get_all_accesses(), "Space Explorer")
+		apply_to_card(I, H, get_syndicate_access(id_access), name, id_icon)
 
+	var/obj/item/device/radio/uplink/U = H.r_store
+	if(istype(U))
+		U.hidden_uplink.uplink_owner = "[H.key]"
+		U.hidden_uplink.uses = uplink_uses
 
+	var/obj/item/device/radio/R = H.l_ear
+	if(istype(R))
+		R.set_frequency(SYND_FREQ)
 
-/datum/outfit/admin/hardsuit
-	back = /obj/item/weapon/tank/jetpack/oxygen
-	mask = /obj/item/clothing/mask/breath
-	id = /obj/item/weapon/card/id
+/datum/outfit/admin/syndicate/infiltrator
+	name = "Syndicate Infiltrator"
 
-/datum/outfit/admin/hardsuit/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	uniform = /obj/item/clothing/under/chameleon
+	glasses = /obj/item/clothing/glasses/hud/security/chameleon
+	shoes = /obj/item/clothing/shoes/syndigaloshes
+	pda = /obj/item/device/pda
+
+/datum/outfit/admin/syndicate/infiltrator/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```

Comment:
This does not include either a dust implant, or an uplink implant, both of which are critical items for SIT team members. It is also missing a flashlight, and possibly other items.